### ISSUE TITLE
rpm: phobos not enabled by default

### ISSUE THREAD

**Issue by thiell** [cea-hpc/coordinatool#2]
Phobos is disabled by default when building rpms, even if the phobos libs are installed.

To build a phobos-aware coordinatool, I added `-Dphobos=enabled` to `coordinatool.spec`:
```
%meson -Dwerror=false -Dphobos=enabled
```

**Comment by courrierg** [comment]
Yes, this is something that was not added to the default specfile as it was more intended as a template. What we can do is add an option --with phobos to rpmbuild to explicitly state that we want to use Phobos. And probably provide this option in the make_rpm script as well.

**Comment by martinetd** [comment]
phobos is now enabled by default ( 01afc6d007c0c0ea31cfae070b9e3b3790469024 ) so the spec also includes it; we can open a new issue if we want to improve packaging later
